lib/Red/Driver/Pg.pm6

```diff
@@ -34,6 +34,22 @@ multi method translate(Red::AST::Select $_, $context?, :$gambi where !*.defined)
     self.Red::Driver::CommonSQL::translate($_, $context, :gambi);
 }
 
+multi method translate(Red::AST::In $_, $context?) {
+    if .right.value ~~ Positional {
```

Comment:
I mean something like:
```perl6
multi method translate(Red::AST::In $_ where .right.value ~~ Positional, $context?) {
```

Reply:
But, instead of testing the right's value type, it would be better to test if it's an Red::AST::Value and if its `.type` is Positional.

```perl6
multi method translate(Red::AST::In $_ where .right ~~ Red::AST::Value && .right.type ~~ Positional, $context?) {
```